### PR TITLE
Surface login/logout operations on app header

### PR DESCRIPTION
**Pull request by msnehagup**

### CODE REVIEW

File: ServerlessLibraryAPI/ClientApp/src/components/Header/AuthControl.js [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+  _onSignoutClick() {
+    this.props.logout(); // clear the redux store before making a call to the backend
+    userService.logout();
+  }
+
+  render() {
+    return (
+      <span>
+        <div className="auth-control" ref={this._menuButtonElement}>
```
Comment (pravagar) [inline ServerlessLibraryAPI/ClientApp/src/components/Header/AuthControl.js]: why is this required?

Comment (msnehagup) [reply ServerlessLibraryAPI/ClientApp/src/components/Header/AuthControl.js]: Good catch! I need to remove this
